packages/backend/src/tokens/interstitial.ts

```diff
@@ -40,6 +41,7 @@ export function addClerkPrefix(str: string | undefined) {
 
 export function loadInterstitialFromLocal(options: Omit<LoadInterstitialOptions, 'apiUrl'>) {
   options.frontendApi = parsePublishableKey(options.publishableKey)?.frontendApi || options.frontendApi || '';
+  const domainOnlyInProd = !isDevOrStagingUrl(options.frontendApi) ? addClerkPrefix(options.domain) : '';
```

Comment:
```suggestion
  const isDomainForProduction = !isDevOrStagingUrl(options.frontendApi) ? addClerkPrefix(options.domain) : '';
```

Reply:
@SokratisVidros  this does return a boolean but a string.

We expect that `domainOnlyInProd` will have a valid value if instance is prod otherwise we assign an empty string. This is on purpose so that the following line works as expected
```typescript
script.src = '${getScriptUrl(proxyUrl || domainOnlyInProd || frontendApi, pkgVersion)}';
```